README.md

```diff
@@ -26,9 +26,12 @@ A collection of our Android templates:
   ```   
     package-name=                      New package name (i.e., com.example.package)
     app-name=                          New app name (i.e., MyApp, "My App", "my-app")
+    template=                          Template type (i.e., xml, compose)
   ```
 
-  Example: `kscript new_project.kts package-name=co.myproject.example app-name="My Project"`
+  Examples: 
+    `kscript new_project.kts package-name=co.myproject.example app-name="My Project" template=xml`
+    `scripts/new_project.kts new_project.kts package-name=co.myproject.example app-name="My Project" template=xml`
```

Comment:
Does this command has to start with `kscript`? 🤔 
![Screenshot 2565-11-29 at 13 56 39](https://user-images.githubusercontent.com/13492460/204460090-846b8bed-1f71-4bfd-b270-52cee3553f69.png)

Reply:
Yes, we need it

Reply:
I mean the below command 😅 It seems to lack the prefix `kscript`
![Screenshot 2565-11-29 at 18 51 50](https://user-images.githubusercontent.com/13492460/204521879-1644948b-058e-415b-8201-931b05ea596e.png)